src/components/Admin/Toolbar.jsx

```diff
@@ -72,7 +75,58 @@ const Toolbar = ({
       );
     }
   };
+  const csvData = objects.map((download) => {
+    const rowData = {};
 
+    if (download.name) {
```

Comment:
Avoid the if statements, this needs to be more abstract as we may add or delete columns so it needs to work out, for now map through the objects entries and use those values directly. Later we need to add an option to delete/hide particular columns (this is a later issue)

Reply:
This will be done by the end of the day